server/app.es6

```diff
@@ -1,26 +1,391 @@
+
+// JSON Validators
+
+let validator = new jsonschema.Validator();
 let validatorOptions = { 'nestedErrors' : true };
 
-var schemas = require('./noncommercial-schema.es6');
-v.addSchema(schemas.noncommercialSchema);
-v.addSchema(schemas.noncommercialApplicantInfoSchema);
-v.addSchema(schemas.noncommercialFieldsSchema);
-v.addSchema(schemas.phoneNumberSchema);
-v.addSchema(schemas.applicantInfoBaseSchema);
-v.addSchema(schemas.commonFieldsSchema);
+let addressSchema = require('./json-schemas/address-schema.es6');
+let applicantInfoBaseSchema = require('./json-schemas/application-info-base-schema.es6');
+let commonFieldsSchema = require('./json-schemas/common-fields-schema.es6');
+let noncommercialApplicantInfoSchema = require('./json-schemas/noncommercial-application-info-schema.es6');
+let noncommercialFieldsSchema = require('./json-schemas/noncommercial-fields-schema.es6');
+let noncommercialSchema = require('./json-schemas/noncommercial-schema.es6');
+let phoneNumberSchema = require('./json-schemas/phone-number-schema.es6');
+let tempOutfitterAppInfoSchema = require('./json-schemas/tempOutfitter-application-info-schema.es6');
+let tempOutfitterFieldsSchema = require('./json-schemas/tempOutfitter-fields-schema.es6');
+let tempOutfitterSchema = require('./json-schemas/tempOutfitter-schema.es6');
+
+validator.addSchema(addressSchema);
+validator.addSchema(applicantInfoBaseSchema);
+validator.addSchema(commonFieldsSchema);
+validator.addSchema(noncommercialApplicantInfoSchema);
+validator.addSchema(noncommercialFieldsSchema);
+validator.addSchema(noncommercialSchema);
+validator.addSchema(phoneNumberSchema);
+validator.addSchema(tempOutfitterAppInfoSchema);
+validator.addSchema(tempOutfitterFieldsSchema);
+validator.addSchema(tempOutfitterSchema);
+
+// Controllers
+
+let sendAcceptedNoncommercialApplicationToMiddleLayer = (application, successCallback, failureCallback) => {
+
+  let authOptions = {
+    url: middleLayerBaseUrl + 'auth',
+    json: true,
+    body: { username: middleLayerUsername, password: middleLayerPassword }
+  };
+
+  let acceptanceOptions = {
+    url: middleLayerBaseUrl + 'permits/applications/special-uses/noncommercial/',
+    headers: { 'x-access-token': undefined },
+    json: true,
+    body: util.translateFromIntakeToMiddleLayer(application)
+  };
+
+  request.post(authOptions, (error, response, body) => {
+    if (error || response.statusCode !== 200) {
+      failureCallback(error || response);
+    } else {
+      acceptanceOptions.headers['x-access-token'] = body.token;
+      request.post(acceptanceOptions, (error, response, body) => {
+        if (error || response.statusCode !== 200) {
+          failureCallback(error || response);
+        } else {
+          successCallback(body);
+        }
+      });
+    }
+  });
+};
+
+// populates an applicationId on the object before return
+let createNoncommercialTempApp = (req, res) => {
```

Comment:
I also wonder if we the individual permit types as there own controller files that pull from shared utilities, like validations etc.